README.md

```diff
@@ -631,7 +631,7 @@ Write code on a whiteboard or paper, not a computer. Test with some sample input
         - [ ] [Supporting Hashing (video)](https://www.lynda.com/Developer-Programming-Foundations-tutorials/Supporting-hashing/149042/177128-4.html)
         - [ ] [Language Support Hash Tables (video)](https://www.lynda.com/Developer-Programming-Foundations-tutorials/Language-support-hash-tables/149042/177129-4.html)
         - [ ] [Core Hash Tables (video)](https://www.coursera.org/learn/data-structures-optimizing-performance/lecture/m7UuP/core-hash-tables)
-        - [ ] [Data Structures (video)](https://www.coursera.org/learn/data-structures/home/week/3)
+        - [ ] [Data Structures (video)](https://www.coursera.org/learn/data-structures/home/week/4)
```

Comment:
The original link goes to hash tables, and this goes to binary search trees.

Reply:
That's strange, because when I am opening the original link it takes me to the playlist where there videos about priority queues, heap sort etc. But, when I go through link with week 4 it takes me to the hash tables :D

Reply:
Weird. I wonder if I see an old version of the course?

Reply:
I think so :)
Didn't know that something like that is possible